### ISSUE TITLE
dynamic commands

### ISSUE THREAD

**Issue by slimshader** [hecomi/uREPL#8]
Hi,

first of all: fantastic asset, I can't believe I've only found it now. Still, better late then never ;)

But to the point: I see that a way to add a command is via attribute which is nice but limits methods to statics. What I would like to have is the ability to add commands at run-time so that for example a single uREPL instance exists during a game but when different scenes are loaded they can add (or remove on unload) commands specific to that scene only.

**Comment by hecomi** [comment]
Thanks for the request! It sounds great. I'll try the implementation.

**Comment by hecomi** [comment]
Released: https://github.com/hecomi/uREPL/releases/tag/v0.4.0

**Comment by slimshader** [comment]
Hi, great to see it implemented but I've noticed that RuntimeCommands is a global static class and ideally I'd like to be able to register dynamic commands per-uREPL instance. Would that be possible?

**Comment by hecomi** [comment]
I'll have to think more about the structure. Please give me some more time to think and implement that.

**Comment by slimshader** [comment]
Of course, take your time, I am glad you'd consider it :) uREPL is grat now anyways.

One more small thing to also add would be to add a bool property to Window class to make it able to start as closed.
